api/src/main/java/io/strimzi/api/kafka/model/nodepool/KafkaNodePoolStatus.java

```diff
@@ -54,6 +55,15 @@ public void setClusterId(String clusterId) {
         this.clusterId = clusterId;
     }
 
+    @Description("The roles currently assigned to this pool.")
```

Comment:
I would be clearer about which roles we are referring to here. Maybe something like this?

```suggestion
    @Description("The KRaft related process roles currently assigned to this pool.")
```

Reply:
Well, they are not limited to KRaft. All ZooKeeper-based brokers have the `broker` role.

Reply:
(Plus the roles are required field even for ZooKeeper-based cluster).

Reply:
Right, so ignore my comment ;-)